### DISCUSSION TITLE
Search Filters for Following Tab

### DISCUSSION THREAD

**Discussion by RetroDestroyer** [modrinth/.github#32] [Feature requests]
**Is your feature request related to a problem? Please describe.**

The following tab is very hard to navigate when the mods are arranged in a seemingly random order. No filters also make it cumbersome to keep scrolling to get a specific pack which may not be a mod. 

**Describe the solution you'd like**

Even simple category splits into mods, modpacks, resource packs, etc. like as they are in user profiles, will be very convenient to use. 

**Describe alternatives you've considered**

A search filter which sorts the tab by "recently opened" or "recently added could also be very useful.

**Additional context**

Just to make the experience of using the following tab better :)

**Comment by triphora** [comment]
Duplicate of #28
